freqtrade/tests/exchange/test_exchange.py

```diff
@@ -737,12 +738,20 @@ def test_get_history(default_conf, mocker, caplog):
 def test_refresh_tickers(mocker, default_conf, caplog) -> None:
     tick = [
         [
-            arrow.utcnow().timestamp * 1000,  # unix timestamp ms
+            (arrow.utcnow().timestamp - 1) * 1000,  # unix timestamp ms
             1,  # open
```

Comment:
Not sure I understand the logic here :/ why do you do -1 ?

Reply:
because i need two ticks in sequence - i could also do -5 or -20 ... does not really matter